### PR TITLE
Updated logo and text to reflect new name

### PR DESCRIPTION
**Pull request by reyg3**

### CODE REVIEW

**Review by lboeman — COMMENTED** [review]
lgtm :+1: :+1: 🙅‍♀️

File: README.md [hunk 1]
```diff
@@ -72,7 +68,7 @@ In addition to our core research applications development team, we've had a numb
     PhD Candidate School of Anthropology</td>
     <td><a href="https://environment.arizona.edu/person/nancy-petersen"><img src="https://environment.arizona.edu/sites/environment.arizona.edu/files/styles/uaqs_medium/public/images/people/NancyP.jpg" width="25"> Nancy Petersen </a>
     <br />
-    AIRES Haury Program</td>
+    AIR Haury Program</td>
```
Comment (tweissaz) [inline README.md:71]: Should we actually use the full name for the Haury program: "Agnese Nelms Haury Program in Environment & Social Justice"

Comment (lboeman) [reply README.md:71]: I agree with Thomas. As he is correct. We should use those words.

**Review by tweissaz — APPROVED** [review]
LGTM 👍👍

**Pull request merged**